packages/frontend/app/home/components/home.tsx

```diff
@@ -0,0 +1,42 @@
+import FeatureItem from "@/components/feature-item";
+import TextAnim from "@/components/text-animation";
+
+export function Home() {
+  // For now just using images from a random watchtower
+  const features = [
+    {
+      label: "Display media for the meeting",
+      img_url:
```

Comment:
this might be a bit of a nit pick but maybe use camelCase here for consistent naming 👍

Reply:
Oh loool I didn't write that. Wasn't that you? 😜 I'll fix it when I can